src/SFA.DAS.ApprenticeFeedback.Api/Controllers/FeedbackTransactionController.cs

```diff
@@ -0,0 +1,41 @@
+﻿
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using MediatR;
+using SFA.DAS.ApprenticeFeedback.Application.Queries.GetEmailTransactions;
+
+
+namespace SFA.DAS.ApprenticeFeedback.Api.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]/")]
+    public class FeedbackTransactionController : Controller
+    {
+        private readonly IMediator _mediator;
+        private readonly ILogger<FeedbackTransactionController> _log;
+
+        public FeedbackTransactionController(IMediator mediator, ILogger<FeedbackTransactionController> log)
+        {
+            _mediator = mediator;
+            _log = log;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> FeedbackTransaction()
```

Comment:
The Api Call is a post to add Feedback Transactions, so it's not a Get Query, it's a Command. `GenerateFeedbackTransactionsCommand` as opposed to a query. The logging is also confusing, your're not retrieving the transactions, you're creating them. 

The catch return null could also be a Bad Request, or 500 status result to be more descriptive an error occured.

Reply:
Changed to command and return 500 in exception handler